test/exports.js

```diff
@@ -8,6 +8,20 @@ describe('exports', function(){
     semver.valid(zmq.version).should.be.ok;
   });
 
+  it('should generate valid curve keypair', function(done) {
+    if (!semver.gte(zmq.version, '4.0.0')) {
+      done();
+      return console.warn('Test requires libzmq >= 4.0.0');
```

Comment:
@szilardhuber, can you also just mention that Test requires libzmq >= 4 "compiled with libsodium"?